scripts/geo.js

```diff
@@ -32,57 +32,52 @@ const getIp = async function(dns){
 }
 
 const getGeo = async function(ip) {
-  let geo
-  await fetch(`http://ip-api.com/json/${ip}`, { headers: { 'accept': 'application/dns-json' } })
-          .then(response => response.json())
-          .then((data) => { geo = data })
-          .catch(err => console.error('./scripts/geo.js', err))
+  let geo;
```

Comment:
This file is no longer an essential script, it was used during early days to create geo-data during pre-build instead of at runtime.